docs/tasks/text-to-image.md

```diff
@@ -178,7 +178,7 @@ all you need to do is use the `WiSEFTCallback`.
 Finetuner will trigger the callback when fine-tuning job finished and merge the weights between the pre-trained model and the fine-tuned model:
 
 ```diff
-from finetuner.callbakcs import WiSEFTCallback
+from finetuner.callback import WiSEFTCallback
```

Comment:
why are we keeping the tasks section?

Reply:
i have removed tasks, wired, seems not reelected in PR